src/odemis/driver/tmcm.py

```diff
@@ -2302,8 +2302,7 @@ def __init__(self, timeout=0, naxes=3, *args, **kwargs):
         # 4 * dict(int -> int: param number -> value)
         self._gstate = [{}, {},
                         # Bank 2: example user config v1
-                        {0: 353304838, 1: 50462723, 2: 17104896, 3: 33555201,
-                         4: 83886082, 5: 196869, 6: 0},
+                        {0: 168034562, 1: 50462723, 2: 17104896},
```

Comment:
Should the default for the kwarg `naxes` now also be `1`?

Reply:
No, it's not directly related. It comes with configuration for 1 axis, but works fine with more axes.